docs/content/docs/ci/Containers.md

```diff
@@ -193,11 +195,25 @@ cd ci/docker
 
 1. Найти новые теги образов.
 См. stdout скрипта или файл `android-emulator/images.txt`
-1. Обнови теги образов в `_main.sh`
+1. Обнови теги образов в build.gradle скриптах.
 
 {{< /tab >}}
 {{< /tabs >}}
 
+#### 2. Залей образы в Docker hub
+
+[hub.docker.com/u/avitotech](https://hub.docker.com/u/avitotech)
+
+1. Залогинься в Docker hub: `docker login --username=avitotech --password=...`
+1. Скачай новый образ из предыдущего шага:\
+`docker pull <DOCKER_REGISTRY>/android/emulator-<API>:<DIGEST>`
+1. Поставь образу такой-же tag, который получился в приватном registry:\
+`docker tag <DIGEST> avitotech/android-emulator-<API>:<DIGEST>`.\
```

Comment:
https://github.com/docker/distribution/issues/1662#issuecomment-213079540

> The image id used within docker can be found in this manifest as config.digest. This config represents the image configuration which will be used within docker. So you could say the manifest is the envelope and image is what is inside. The manifest digest will always be different than the image id BUT for any given manifest the same image id should always be produced. Since it is a hash chain, **we cannot guarantee that the manifest digest will always be the same for a given image id**. In most cases it should usually produce the same digest, we just cannot guarantee it but do a best effort.